clients/py/README.md

```diff
@@ -20,6 +20,8 @@ w3.middleware_onion.add(construct_sign_and_send_raw_middleware(account))
 
 # Finally, wrap the provider to add Sapphire end-to-end encryption
 w3 = sapphire.wrap(w3)
+# Optionally, select the gas price oracle implemented in the Oasis Web3 Gateway
```

Comment:
```suggestion
# Optionally, select the gas price oracle implemented in the Oasis Web3 Gateway.
```

Also, please add punctuation above.

Reply:
Hvala. I probably should run a commit hook 😬